cmd/permission/main.go

```diff
@@ -2,24 +2,23 @@ package main
 
 import (
 	"fmt"
+
+	"github.com/openslides/openslides-permission-service/internal/permission"
 )
 
 type fakeDataProvider struct{}
 
-type Fqfield = string
-type Value = interface{}
```

Comment:
The `=` is wrong. A type statement with a `=` is a type alias. This is a special construct needed for backswords compatibly. You will probably not need it soon. The correct way is `type NAME BASETYPE` (without the `=`).

But in your case you do not need the type here. The type you need is `permission.Fqfield` which is a different type then `Fqfield` defined here.